core/src/main/java/org/infinispan/protostream/impl/parser/AnnotationParser.java

```diff
@@ -121,6 +121,14 @@ private Map<String, AnnotationElement.Attribute> parseAttributes() {
                   return members;
                }
             }
+            case LBRACE: {
```

Comment:
don't know why this is required

Reply:
This is to support default annotation values as arrays, e.g.
```java
@MyAnnotation({"a","b","c"})
```
I noticed it was breaking in my experiments

Reply:
it would be nice to have a test in `AnnotationParserTest` with that format.

Reply:
I've added a test for this